packages/terra-framework-site/src/app/App.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { HashRouter as Router, Route } from 'react-router-dom';
+import Base from 'terra-base';
+import ThemeProvider from 'terra-theme-provider';
+import NavigationLayout from './common/navigation/NavigationLayout';
+import ContentWrapper from './common/navigation/wrappers/ContentWrapper';
+import MenuWrapper from './common/navigation/wrappers/MenuWrapper';
+
+import ApplicationHeader from './ApplicationHeader';
+import './App.scss';
+
+const propTypes = {
+  config: PropTypes.object,
+};
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+const themes = {
+  'Default Theme': '',
+  'Consumer Theme': 'cerner-consumer-theme',
```

Comment:
Probably don't need consumer theme introduced here

Reply:
Word, I'll remove it.